extensions/2.0/Khronos/KHR_mesh_instancing/samples/teapots_galore_TRS/teapots_galore_TRS.gltf

```diff
@@ -0,0 +1,565 @@
+{
+    "accessors": [
+        {
+            "bufferView": 0,
+            "componentType": 5126,
+            "count": 12096,
+            "extras": {
+                "arrayIndex": 0
+            },
+            "max": [
+                3.433150053024292,
+                3.1500000953674316,
+                2.0
+            ],
+            "min": [
+                -3.0,
+                0.0,
+                -2.0
+            ],
+            "type": "VEC3"
+        },
+        {
+            "bufferView": 1,
+            "componentType": 5126,
+            "count": 12096,
+            "extras": {
+                "arrayIndex": 1
+            },
+            "type": "VEC3"
+        },
+        {
+            "bufferView": 2,
+            "componentType": 5125,
+            "count": 12096,
+            "extras": {
+                "arrayIndex": 2
+            },
+            "type": "SCALAR"
+        },
+        {
+            "bufferView": 3,
+            "componentType": 5126,
+            "count": 958,
+            "extras": {
+                "arrayIndex": 3
+            },
+            "type": "VEC3"
+        },
+        {
+            "bufferView": 4,
+            "componentType": 5126,
+            "count": 958,
+            "extras": {
+                "arrayIndex": 4
+            },
+            "type": "VEC3"
```

Comment:
This is our accessor for the instance rotation buffer, right?
Can we change this to be a Quaternion vec4 instead of an Euler angle vec3?

Reply:
We definitely can change Euler angles to a Quaternion.  Is that what most engines would expect?

Reply:
I think so, yes — this would match the quaternion rotations we store on nodes: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#transformations